src/PeerStore/PeerStore.ts

```diff
@@ -0,0 +1,56 @@
+import PeerInfo from "@polykey/PeerStore/PeerInfo"
+
+/**
+ * Responsible for managing known peers, as well as their addresses and metadata
+ */
+class PeerStore {
```

Comment:
If this is sort of modelled off libp2p, then our peerstore and similar modules can be separate libraries, and even libp2p libraries like multiaddr and multihash.

BTW have you looked at our hs-multiaddr and hs-multihash?

Reply:
I have taken a look at them as yet, but yeah I think PeerStore could be renamed to something like `PubKeyStore` or `js-pubkeystore`

Reply:
As discussed elsewhere, this will stay inside the polykey library repo